src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java

```diff
@@ -0,0 +1,19 @@
+package api.java.openiss.ws.soap.service;
+
+import javax.jws.WebMethod;  
+import javax.jws.WebService;  
+import javax.jws.soap.SOAPBinding;  
+import javax.jws.soap.SOAPBinding.Style;  
+
+/**
+ * Service Endpoint Interface
+ *
+ * @author zephi
+ */
+
+@WebService  
+@SOAPBinding(style = Style.DOCUMENT)  
+public interface OpenISSSOAPService {
+    @WebMethod String fetchXML(String url, String reqType);  
```

Comment:
(D) `fetchXML()` was not a web method in spec, why is it here? Why is it needed?

Reply:
checked (D) has been resolved, remove `fetchXML()`.